packages/leva/src/components/Folder/Folder.tsx

```diff
@@ -33,6 +33,7 @@ export const Folder = React.memo(({ name, parent, tree, isRoot = false, collapse
       firstRender.current = false
       return
     }
+    console.log('toggle')
```

Comment:
☝️

Reply:
Dammit, sorry. We need to add a lint rule 😅

Reply:
I'm ashamed of how many useless commits I'm making atm sorry I'm a bit tired I guess.